app/konnect/getting-started/configure-mesh.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Configure a Global Control Plane with Kong Mesh
+content_type: tutorial
+beta: true
+---
+
+Using Mesh Manager, you can create {{site.mesh_product_name}} global control planes to manage your {{site.konnect_saas}} mesh services. 
+
+A global control plane is a managed central component that stores and distributes (to local zones) all of the configuration and policies for your meshes and services. The global control planes are responsible for validating and accepting connections from local zone control planes and distributing the appropriate configuration down to each local zone as required. They also serve as the target for all `kumactl` CLI operations when manipulating resources and configuration within the mesh deployment.
+
+![mesh global control plane](/assets/images/diagrams/gslides/kuma_multizone.svg)
+
+> _**Figure 1:** {{site.mesh_product_name}} can support multiple zones (like a Kubernetes cluster, VPC, data center, etc.) together in the same distributed deployment. Then, you can create multiple isolated virtual meshes with the same control plane in order to support every team and application in the organization._
+
+For the purposes of this guide, you'll use {{site.mesh_product_name}} as your service mesh to create a global control plane. You will use Kubernetes to install various service mesh components.
+
+## Prerequisites
+
+* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
+* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [Download the latest version of {{site.mesh_product_name}}](/mesh/latest/production/install-kumactl/)
+
+## Create a global control plane in {{site.konnect_short_name}}
+ 
+1. From the left navigation menu in {{site.konnect_short_name}}, open {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager).
+1. Click **New Control Plane**.
+1. Enter "example-cp" in the **Name** field.
+1. Click **Save**.
+
+You now have a {{site.mesh_product_name}} global control plane. This control plane can't do anything at the moment because we need to connect a zone to it next. 
+
+## Create a zone in the global control plane
+
+After creating the global control plane, you must add a zone to that control plane. Adding a zone allows you to manage services added to that zone and send and receive configuration changes to the zone. 
+
+1. In {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager), click the `example-cp` control plane you just created, and then click **Zones** in the sidebar.
+1. Click **Create Zone**. 
+1. Enter "eu" in the **Name** field for the new zone, and then click **Create Zone & generate token**. 
+    {:.note}
+    > **Note:** The zone name must consist of lower case alphanumeric characters, `.`, or `-`. It must also start and end with an alphanumeric character.
```

Comment:
@jakubdyszkiewicz is '.' actually allowed?

Reply:
It should not be allowed